docs/io/optional/logging_configuration.ipynb

```diff
@@ -0,0 +1,512 @@
+   "id": "df50d2e1",
+   "metadata": {},
+   "source": [
+    "The `specific_logging` parameter in the `debug` section of the schema is similar in functionality to the `specific` functional argument. It takes **Boolean** values, i.e., `True` or `False`. It is an optional parameter. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b71946ad",
+   "metadata": {},
+   "source": [
+    "Let us see an example when we set `specific_logging` to `True` at the `Debug` level."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "64ce1048",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config[\"debug\"][\"logging_level\"] = \"Debug\"\n",
+    "config[\"debug\"][\"specific_logging\"] = True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "33696bdd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sim = run_tardis(config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3e895aec",
+   "metadata": {},
+   "source": [
+    "Setting `specific_logging` to `False` to check the actual output at the `DEBUG` level for the simulation logs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "762c858f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config[\"debug\"][\"specific_logging\"] = False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "581f2d8d",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "sim = run_tardis(config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e8f1dbe",
+   "metadata": {},
+   "source": [
+    "### What Happens When Both Parameters Are Specified? { Function & YAML Arguments }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f32403c7",
+   "metadata": {},
+   "source": [
+    "If a user specifies both the parameters passed through the `log_state` & `logging_level` in the **YAML** configuration file, then the `log_state` parameter takes precedence & is used to determine the logging level for the simulation logs."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f2a24e11",
+   "metadata": {},
+   "source": [
+    "Let us consider the following example for the configuration:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "863042a4",
+   "metadata": {},
+   "source": [
+    "Continuing from the previous example, the `config[\"debug\"][\"logging_level\"]` is set to `Debug` via the **YAML** file. The user will also set the `log_state` to `Info`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "30763e52",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "sim = run_tardis(config, log_state = \"Info\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "674f4972",
+   "metadata": {},
+   "source": [
+    "A new message can be seen from the execution of the simulation, that there is a new message,\n",
```

Comment:
remove the " that there is a new message,", as it repeats itself currently.

Reply:
If you want, you could then remove the final comma and replace it with a colon, and then capitalize the first work on line 467 as another possible syntax route.